### PR TITLE
imp(jssdk): export type PhactoryAPI

### PR DESCRIPTION
**Pull request by Leechael**

### CODE REVIEW

File: frontend/packages/sdk/src/index.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export * from './pruntime/eip712'
 export { default as createPruntimeClient } from './pruntime/createPruntimeClient'
 export { default as signAndSend, SignAndSendError } from './utils/signAndSend'
 
-export const PhactoryAPI = pruntime_rpc.PhactoryAPI
 export const pruntimeRpc = pruntime_rpc
+export type PhactoryAPI = pruntime_rpc.PhactoryAPI
+export const PhactoryAPI = pruntime_rpc.PhactoryAPI
```
Comment (kvinwang) [inline frontend/packages/sdk/src/index.ts:19]: What's the purpose of "export `const` PhactoryAPI"?

Comment (Leechael) [reply frontend/packages/sdk/src/index.ts:19]: The `export type` statement exports only the type definition, while the `export const PhactoryAPI` statement exports both the type definition and the code implementation.

**Review by kvinwang — APPROVED** [review]
LGTM

File: frontend/packages/sdk/src/index.ts (hunk 1 above)
Comment (kvinwang) [reply frontend/packages/sdk/src/index.ts:19]: Got it. Isn't there a one-line syntax to reexport the class?

**Pull request merged**